src/components/HeroSection.vue

```diff
@@ -20,7 +20,7 @@
       </div>
       <div class="is-hidden-desktop">
         <input required="required"
-                autofocus="true"
+                v-focus
```

Comment:
I think it would be better to not auto focus on mobile because it would cause the keyboard to pop up without the user's "permission" which is generally not a good thing. It's safe to assume the intent, but not good to change what's going on on screen without an action triggering it.

Reply:
Definitely, would make changes